README.md

```diff
@@ -50,7 +57,7 @@ The master branch is available at https://live.door43.org (soon to be at https:/
 
 #### Pre Production Testing
 
-You may run `make test`, or `make build`, or `make serve` to test and review your changes locally.  Once the `cibuild.sh` script passes successfully locally, you may commit and push to the `develop` branch.  You can do this by running `make commit`.
+You may run `make test`, or `make build`, or `make serve` to test and review your changes locally.  Once the `cibuild.sh` script passes successfully locally, push `test` branch.  Follow setup instructions in `s3_test_push.sh` and then run the script  `s3_test_push.sh`.
```

Comment:
@richmahn - Not sure what you are asking here, but this section is titled pre-production testing.  As I understand we are now doing that on `test`.  And pushing to that is now done by script.

Are you wanting a note added that after they make a PR against develop branch and it has been merged, that Travis will automatically deploy to dev for further testing?